src/components/modes/teacher/BackgroundToggle.tsx

```diff
@@ -134,11 +140,13 @@ const BackgroundToggle = () => {
               onChange={handleToggle}
             />
           }
+          label={undefined}
```

Comment:
Is it useful?

Reply:
Actually, I changed the component. I don't remember why I have a FormControlLabel but I suspect it is legacy code that I should have removed.